src/main/java/io/opentracing/contrib/spanmanager/tracer/ManagedSpanBuilder.java

```diff
@@ -107,6 +115,16 @@ public SpanBuilder withStartTimestamp(long microseconds) {
         return rewrap(delegate.withStartTimestamp(microseconds));
     }
 
+    @Override
+    public ActiveSpan startActive() {
+        return delegate.startActive();
+    }
+
+    @Override
+    public Span startManual() {
+        return delegate.startManual();
+    }
+
```

Comment:
Not 'solved' by simple delegation I'm afraid.
The `ActiveSpan` concept introduced in API v0.30.0 _replaces_ the `SpanManager` concept.
You can view the spanmanager lib as a precursor of something 'official' in the API.
This is the reason it hasn't been _upgraded_ to the new API version yet.